VAMobile/documentation/docs/Engineering/FrontEnd/PushNotifications/DeepLinking.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Deep Linking
+---
+
+# Deep Linking
+
+## Overview
+
+Deep links in the app are handled by the React Navigation library. The React Navigation linking configuration handles mapping links to the appropriate screens, and updating the navigation state when necessary.
+
+## Setting up push notifications
+
+A prerequisite for adding deep linking support for a feature's push notification is to have push notifications setup for that feature. An architectural overview of how push notifications work in the app and information on how to add them for a feature can be found [here](../../BackEnd/Features/PushNotifications.md).
+
+In order for deep linking to work for your push notification, the `url` field must be present in the push template. The URL should be prefixed with `vamobile://` and followed by the path of the deep link you want to add, e.g. `vamobile://messages/%MSG_ID%`. In this example, the path of the deep link is prefixed with `vamobile://`, starts with `messages`, and is followed by the `MSG_ID` parameter, which is a field that will be populated from the API call that triggers the push notification. This will result in push notifications returning a payload that looks some like:
+
```

Comment:
'looks some like:' maybe 'looks something like:' ?

Reply:
Good catch, just fixed this typo